azure_pipelines/templates/tests-with-conf-file.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+  schema: 'MSAL Test Automation (iOS)'
+  build: 'MSAL Test Automation (iOS)'
+
+steps:
+  - checkout: self
+    clean: true
+    submodules: true
+    fetchDepth: 1
+    persistCredentials: true
+    path: s
+    
+  - task: Bash@3
+    displayName: 'Select Xcode version'
+    inputs:
+      targetType: 'inline'
+      script: '/bin/bash -c "sudo xcode-select -s /Applications/Xcode_15.0.1.app"'
+  
+  - task: Bash@3
+    displayName: Go to project folder
+    inputs:
+      targetType: 'inline'
+      script: |        
+        cd $(Agent.BuildDirectory)/s
+  
+  - script: 'gem install xcpretty'
+    displayName: 'Install xcpretty'
+
+  - task: AzureCLI@2
+    displayName: 'Read configuration from KeyVault'
+    inputs:
+      azureSubscription: 'AuthSdkResourceManager'
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: 'python get_automation_conf_file.py'
+      workingDirectory: azure_pipelines/scripts/
+  - task: Bash@3
+    displayName: Build for testing
+    inputs:
+      targetType: 'inline'
+      script: |
+        mv azure_pipelines/scripts/conf.json MSAL/test/automation/conf.json
+        ruby TestRubyFile.rb
+        xcodebuild build-for-testing \
+          -workspace MSAL.xcworkspace \
+          -scheme '${{ parameters.schema }}' \
+          -sdk iphonesimulator \
+          -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.2' \
+          -derivedDataPath 'build' \
+          | tee xcodebuild.log \
+          | xcpretty -c
+
+  - task: Bash@3
+    displayName: Run automations
+    inputs:
+      targetType: 'inline'
+      script: |
+        ls build/Build/Products/
+        xcodebuild test-without-building \
+            -xctestrun 'build/Build/Products/${{ parameters.build }}_iphonesimulator17.0-x86_64.xctestrun' \
+            -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.2' \
+            -retry-tests-on-failure \
+            -parallel-testing-enabled NO \
```

Comment:
I thought we had enabled it to run on parallel simulators. Does it not work faster in that way?